### PR TITLE
pgcli 3.5.0

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by branchvincent** [comment]
Failing to find our `libpq` ([source](https://github.com/psycopg/psycopg/blob/052534ff0c8d947ae4d3b97a655c3d643ff8873a/psycopg/psycopg/pq/misc.py#L55-L68)):
```
ImportError: no pq wrapper available.
  Attempts made:
  - couldn't import psycopg 'c' implementation: No module named 'psycopg_c'
  - couldn't import psycopg 'binary' implementation: No module named 'psycopg_binary'
  - couldn't import psycopg 'python' implementation: libpq library not found
```

Related: https://github.com/psycopg/psycopg/issues/364, https://github.com/orgs/Homebrew/discussions/3595

File: Formula/pgcli.rb [hunk 1]
```diff
@@ -92,12 +96,18 @@ class Pgcli < Formula
   def install
     venv = virtualenv_create(libexec, "python3.10")
 
+    resource("psycopg").stage do
+      inreplace "psycopg/pq/_pq_ctypes.py", "libname = find_libpq_full_path()",
+                                            "libname = '#{Formula["libpq"].opt_lib/shared_library("libpq")}'"
```
Comment (carlocab) [inline Formula/pgcli.rb:101]: I think this should be using `pg_config` to find `libpq` automatically -- is that not working?

File: Formula/pgcli.rb [hunk 2]
```diff
@@ -92,12 +96,18 @@ class Pgcli < Formula
...
+      venv.pip_install Pathname.pwd
+    end
+
     resource("pytzdata").stage do
       system Formula["poetry"].opt_bin/"poetry", "build", "--format", "wheel", "--verbose", "--no-interaction"
       venv.pip_install Dir["dist/pytzdata-*.whl"].first
```
Comment (carlocab) [inline Formula/pgcli.rb:107]:
```suggestion
      venv.pip_install Pathname.glob("dist/pytzdata-*.whl").first
```

File: Formula/pgcli.rb [hunk 3]
```diff
@@ -92,12 +96,18 @@ class Pgcli < Formula
...
     end
 
-    venv.pip_install resources.reject { |r| r.name == "pytzdata" }
+    venv.pip_install resources.reject { |r| r.name == "psycopg" || r.name == "pytzdata" }
```
Comment (carlocab) [inline Formula/pgcli.rb:110]:
```suggestion
    venv.pip_install resources.reject { |r| %w[psycopg pytzdata].include? r.name }
```

File: Formula/pgcli.rb (hunk 1 above)
Comment (branchvincent) [reply Formula/pgcli.rb:101]: It does work when `pg_config` is on `PATH`, but `libpq` is keg only

Comment (carlocab) [reply Formula/pgcli.rb:101]: Right, we should add a comment documenting this. I think we may make `libpq` not-keg-only.

**Review by carlocab — APPROVED** [review]

File: Formula/pgcli.rb (hunk 1 above)
Comment (branchvincent) [reply Formula/pgcli.rb:101]: Good idea, #111006

**Pull request closed without merging**